src/openepd/model/org.py

```diff
@@ -71,6 +71,12 @@ class Plant(WithAttachmentsMixin, WithAltIdsMixin, BaseOpenEpdSchema):
         example="865P2W3V+3W.interface.com",
         default=None,
     )
+    pluscode: str | None = pyd.Field(
+        default=None,
+        description="Plus code (aka Open Location Code) of plant's location",
+        deprecated="Pluscode field is deprecated. If users need a pluscode they can obtain it from "
```

Comment:
I don't see `deprecated` in the args of `pyd.Field` function. Is this correct?

Reply:
You are right - it appeared in pydantic 2, now as I checked. I'll leave it there though because a) it does not break things b) it will be preserved on migration. 

I will add also deprecation warning into the description itself.